docs/development/contributing.md

```diff
@@ -40,14 +40,15 @@ mail to the [Technical Board of the ILIAS society](mailto:tb@lists.ilias.de).
 ### Pull Request to the Repositories
 
 Pull requests (PRs) will be assigned by the [Technical Board
-(TB)](http://www.ilias.de/docu/goto.php?target=grp_5089&client_id=docu) to the
-responsible maintainer. The TB will also help to resolve problems with PRs and
-associated processes, if you require mediation please leave a mention via
-`@ILIAS-eLearning/technical-board` in the discussion of the PR.
-
-Please make sure to understand that the ILIAS society has a [process for
-functional feature request](https://www.ilias.de/docu/goto_docu_wiki_wpage_788_1357.html) 
-before starting to create your PR. Your PR should thus only contain bug fixes or 
+(TB)](https://docu.ilias.de/goto.php?target=grp_5089&client_id=docu) to the
+appropriate maintainer. The TB will also help to resolve problems with PRs and
```

Comment:
Is maintainer suitable here?

Reply:
What about “project member”, or “person”?

Reply:
I went with "project member". It could also be "developer", but as I could imagine processes in which the primary assignee for PRs wouldn't be a developer herself ...